### DISCUSSION TITLE
How can I get buyer-tax-registration-id?

### DISCUSSION THREAD

**Discussion by JosemiMoran** [abuzuhri/Amazon-SP-API-CSharp#579] [Q&A]
Hi, I need help because I want to get the "buyer-tax-registration-id" when the order is created (only if it is Business), but I only found reports that give me that id when the order is shipped, is there any way to get that id without marking the order as shipped?
When I use getOrderBuyerInfo the buyerTaxInformation is null and I have all the necessary permissions accepted. I've been stuck on this for weeks and need some help.
Many thanks in advance!

**Comment by cvalerianotooeasy** [comment]
Were you able to resolve the problem? I'm in the same situation as you but it still seems that when I request buyerTaxInformation the Amazon system responds that I don't have permissions!
